source/common/protobuf/utility.cc

```diff
@@ -209,4 +209,14 @@ uint64_t DurationUtil::durationToSeconds(const ProtobufWkt::Duration& duration)
   return Protobuf::util::TimeUtil::DurationToSeconds(duration);
 }
 
+void TimestampUtil::systemClockToTimestamp(const SystemTime system_clock_time,
```

Comment:
Can you add a dedicated test for this?

Reply:
Done in `/test/common/protobuf/utility_test.cc`.